cob_default_robot_config/cob3-2/sound_configurations.yaml

```diff
@@ -0,0 +1,41 @@
+speech_mode: FEST_EN
+#wav_de_path: /home/cob/svn/cob/Files/Audio/WiMi/wav_de/
+wav_en_path: /home/cob/svn/cob/Files/Audio/WiMi/wav_de/
+temp_wav_file: /home/fmw/wav/swift_temp.wav
```

Comment:
no idea... I copied all the configuration files from cob3-2 backup, and I adapted it... but I think that we can remove these lines